src/gpuarray_elemwise.c

```diff
@@ -321,20 +323,49 @@ static int check_basic(GpuElemwise *ge, void **args, int flags,
   n = 1;
   for (j = 0; j < nd; j++) {
     p = 0;
+    read_write_arrays_found = 0;
```

Comment:
The exact number of arrays doesn't matter, so I would make this a flag like `dim_fixed` or something of the sort.